### PR TITLE
@alloy => Add some metadata stories, and fix artwork brick/grid items

### PR DESCRIPTION
**Pull request by mzikherman**
In adding a few stories for artwork metadata (goal is to have a 'style guide' of all the cases), I noticed that we were essentially, suppressing sale and contact messages by default. I commented in https://github.com/artsy/reaction/pull/73#discussion_r147177887 , but it seems like it's a mismatch of props/defaults , so a pretty simple fix. I'll point it out inline.

### CODE REVIEW

File: src/Components/Artwork/GridItem.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ export class ArtworkGridItem extends React.Component<Props, null> {
             style={{ position: "absolute", right: "10px", bottom: "10px" }}
           />
         </Placeholder>
-        <ArtworkMetadata artwork={artwork} />
+        <ArtworkMetadata extended artwork={artwork} />
```
Comment (mzikherman) [inline src/Components/Artwork/GridItem.tsx]: So, the `GridItem` is used in the `ArtworkGrid`, and thus is used in the artwork filters. I don't know why we wouldn't want to display the sale messaging, so this passes in true. This should fix the gene page.

File: src/Components/__stories__/ArtworkMetadata.story.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <Relay.RootContainer
+      renderFetched={ArtworkMetadataGenerator}
+      Component={ArtworkMetadata}
+      route={new ArtworkQueryConfig({ artworkID: props.artworkID })}
+    />
+  )
+}
+
+storiesOf("Components/Artwork/Metadata", module)
+  .add("A NFS artwork", () => <ArtworkExample artworkID="andy-warhol-skull" />)
```
Comment (mzikherman) [inline src/Components/__stories__/ArtworkMetadata.story.tsx]: Goal is to add a lot of cases here (probably separated by partner)? It should be pretty quick to add. So far these are looking good!

screen shot 2017-10-26 at 12 32 51 pm

File: src/Components/Artwork/GridItem.tsx (hunk 1 above)
Comment (alloy) [reply src/Components/Artwork/GridItem.tsx]: Do we even really need the `extended` option? If not, then I’d say let’s get rid of it so nobody else uses the component not realising they might need to specify this.

File: src/Components/__stories__/ArtworkMetadata.story.tsx (hunk 2 above)
Comment (alloy) [reply src/Components/__stories__/ArtworkMetadata.story.tsx]: I’m not a big fan of shortening text, especially not when used in a UI. Was there a reason to do that here (“NFS”) other than not having to type more?

**Comment by alloy** [comment]
Would you mind rebasing this on my relay-modern branch and making a PR to merge it into that branch?

File: src/Components/__stories__/ArtworkMetadata.story.tsx (hunk 2 above)
Comment (mzikherman) [reply src/Components/__stories__/ArtworkMetadata.story.tsx]: Nope, I guess I've seen NFS used sometimes as an abbreviation (at Artsy), but no special reason whatsoever.

File: src/Components/Artwork/GridItem.tsx (hunk 1 above)
Comment (mzikherman) [reply src/Components/Artwork/GridItem.tsx]: Yea, we do need it- basically for the loyalty artworks we wanted the ability to use these components but hide the sale lines (as well as some other tweaks). So `extended` was used to signify that.

I think, if anything, it probably makes more sense for `extended=true` to be the default however, and override it in the loyalty artwork component.

Comment (alloy) [reply src/Components/Artwork/GridItem.tsx]: Agreed 👍

**Pull request closed without merging**